### DISCUSSION TITLE
Is it worth buying Colab Pro ($9.99) plan for using SD?

### DISCUSSION THREAD

**Discussion by i-huzaifa-arshad** [TheLastBen/fast-stable-diffusion#2632] [Q&A]

**Comment by TheLastBen** [comment]
go for Paperspace or Runpod instead, it's more stable and cheaper than google

**Reply by i-huzaifa-arshad** [reply]
Which one would you suggest? Paperspace or Runpod? In terms of easyness and cheap?

**Reply by TheLastBen** [reply]
if you want pay as you go, Runpod has a lot of options, if you want subscription, Paperspace is better for that

**Reply by i-huzaifa-arshad** [reply]
Ok cool. Thank you so much sir.
